examples/grommet/src/widgetToolbarItems/NumberWidget.tsx

```diff
@@ -0,0 +1,18 @@
+import { type ShowCaseWidgetProps } from '@openfun/verna';
+import { defineMessages, useIntl } from 'react-intl';
+import WidgetWrapper from './WidgetWrapper';
+
+const messages = defineMessages({
+  NumberWidget: {
+    defaultMessage: 'Number answer poll',
```

Comment:
```suggestion
    defaultMessage: 'Number of poll answers',
```

Reply:
It's not for the number of poll answers, it's an input of type number, i had no clue how to name it D: if you've an idea i take it